src/__tests__/stubs/moby-epub3-absolute-hrefs.ts

```diff
@@ -610,15 +618,6 @@ const mobyEpub3Manifest = {
       type: 'application/vnd.ms-opentype',
       href: 'http://remote.dev/samples/moby-epub3-exploded/OPS/fonts/STIXGeneralBolIta.otf',
     },
-    {
-      type: 'application/xhtml+xml',
-      rel: 'contents',
```

Comment:
It looks like we are losing the `rel: 'contents'` when we place it in the reading order. Is there an easy way to keep that?